_includes/manuals/1.17/1.2_release_notes.md

```diff
@@ -0,0 +1,225 @@
+ * cache notification drawer state ([#20485](http://projects.theforeman.org/issues/20485))
+ * Allow to mass mark notification group as read ([#19192](http://projects.theforeman.org/issues/19192))
+ * Use Dynflow as an ActiveJob backend ([#18618](http://projects.theforeman.org/issues/18618))
+
+#### Orchestration
+ * Add a setting that allows the user to enable orchestration when puppet facts are uploaded. ([#21565](http://projects.theforeman.org/issues/21565))
+ * Add VM boot failure faster feedback loop for openstack ([#19459](http://projects.theforeman.org/issues/19459))
+
+#### Organizations and Locations
+ * Add host count to location API call ([#21012](http://projects.theforeman.org/issues/21012))
+
+#### Plugins-Engines
+ * Pagelets - add tab extension point for hostgroup edit page ([#21658](http://projects.theforeman.org/issues/21658))
+ * Pagelets - add extension point for host show page ([#21579](http://projects.theforeman.org/issues/21579))
+ * Ability for engine/plugin to extend RABL responses ([#3763](http://projects.theforeman.org/issues/3763))
+ * Allow plugins webpack code to be used ([#20097](http://projects.theforeman.org/issues/20097))
+ * Add an option to create plugin migrations with properly marked scope ([#19589](http://projects.theforeman.org/issues/19589))
+ * Lock plugin roles ([#19039](http://projects.theforeman.org/issues/19039))
+
+#### Rails
+ * Add support for running under Ruby on Rails 5.0 ([#19050](http://projects.theforeman.org/issues/19050))
+
+#### Reporting
+ * Add scoped_search on host id for reports ([#21602](http://projects.theforeman.org/issues/21602))
+ * Unify the way how we display dates ([#19047](http://projects.theforeman.org/issues/19047))
+
+#### Search
+ * Filter roles by assigned permissions in UI ([#20047](http://projects.theforeman.org/issues/20047))
+ * Add the ability to search for a puppet class in the configuration groups ([#7499](http://projects.theforeman.org/issues/7499))
+
+#### Security
+ * Protection from Brute Force Password Attacks  ([#4238](http://projects.theforeman.org/issues/4238))
+
+#### Settings
+ * settings page should show when non-default settings are selected ([#21238](http://projects.theforeman.org/issues/21238))
+ * Global HTTP proxy for outgoing requests ([#20201](http://projects.theforeman.org/issues/20201))
+
+#### Smart Variables
+ * Separate Smart variables and Smart parameters in Host/HostGroup -> Parameters tab. ([#12829](http://projects.theforeman.org/issues/12829))
+ * Allow searching smart class parameters by environment ([#20083](http://projects.theforeman.org/issues/20083))
+ * Prioritize attribute order in puppet classes limited to 255 chars ([#19934](http://projects.theforeman.org/issues/19934))
+ * Inform user that all Puppet Class sub entities affect saving the whole puppet class ([#19582](http://projects.theforeman.org/issues/19582))
+
+#### Smart proxies
+ * Support SSHFP dns records ([#20370](http://projects.theforeman.org/issues/20370))
```

Comment:
This was closed, not resolved. https://github.com/theforeman/smart-proxy/pull/536 is still open